WordPress/Classes/Networking/SharingServiceRemote.swift

```diff
@@ -212,25 +212,76 @@ public class SharingServiceRemote : ServiceRemoteREST
     /// Update the shared status of the specified publicize connection
     ///
     /// - Parameters:
-    ///     - shared: True if the connection is shared with all users of the blog. False otherwise.
     ///     - connectionID: The ID of the publicize connection.
+    ///     - externalID: The connection's externalID. Pass `nil` if the keyring
+    /// connection's default external ID should be used.  Otherwise pass the external
+    /// ID of one if the keyring connection's `additionalExternalUsers`.
+    ///     - siteID: The WordPress.com ID of the site.
+    ///      -success: An optional success block accepting no arguments.
+    ///     - failure: An optional failure block accepting an `NSError` argument.
+    ///
+    public func updatePublicizeConnectionWithID(connectionID: NSNumber,
+        externalID: String?,
+        forSite siteID: NSNumber,
+        success: (RemotePublicizeConnection -> Void)?,
+        failure: (NSError! -> Void)?) {
+            let endpoint = "sites/\(siteID)/publicize-connections/\(connectionID)"
+            let path = self.pathForEndpoint(endpoint, withVersion: ServiceRemoteRESTApiVersion_1_1)
+            let externalUserID = (externalID == nil) ? "false" : externalID!
+
+            let parameters = [
+                PublicizeConnectionParams.externalUserID : externalUserID
+            ]
+
+            api.POST(path,
+                parameters: parameters,
+                success: { (operation: AFHTTPRequestOperation!, response: AnyObject!) in
+                    guard let onSuccess = success else {
+                        return
+                    }
+
+                    let dict = response as! NSDictionary
```

Comment:
Nipticky comment. Maybe `as? NSDictionary`, and we move that one into the guard check few lines up?